src/components/WeatherPanes/ListPane.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from 'assets/css/styles.css';
+import Store from 'Store';
+
+import Icon from 'components/reusable/icons/Icon';
+
+class ListPane extends React.PureComponent {
+  constructor(props){
+    super(props);
+    this.state = {}
+  }
+
+  componentDidMount() {
+    console.log('pep')
+    this.getTempUnit();
+  }
+
+  // componentWillUnmount() {
+  // }
+
+  getTempUnit() {
+    Store.subscribe('SET_TEMP_UNIT', () => {
+      console.log(Store.getProp('tempUnit'))
+      this.setState({
```

Comment:
optionally you can lift this state up to Parent component and pass it down to ListPane as a prop... what do you think ?